tests/mowgli_etl_test/pipeline/wdc/wdc_heuristic_product_type_classifier_test.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+from json import loads
+
+from mowgli_etl.pipeline.wdc.wdc_heuristic_product_type_classifier import WdcHeuristicProductTypeClassifier as WdcHPTC
+
+#Should have Path argument, for now using clean file already in directory
+def test_heuristic_prdocut_type_classifier():
+    with open("offers_corpus_english_v2_random_100_clean.jsonl","r") as data:
+        item_counter=0
+        for row in data:
+            item_counter += 1
+            information = loads(row)
+            listing = information["title"]
```

Comment:
This would all be nicer with the input model I mentioned. We can discuss that later.